.eslintrc.cjs

```diff
@@ -7,11 +7,21 @@ module.exports = {
   },
   /** Default rules for Javascript, Typescript & Tests */
   rules: {
+    // Use const/let
     'no-var': 'error',
+
+    /**
+     *  Force `===` but allow `== null`
+     *  to prevent `x === undefined || x === null`
```

Comment:
Why do we want to allow `== null`? Isn't it extremely rare for a value to truly be either `undefined` or `null`?

Reply:
most libraries are really bad at returning just one of `null` or `undefined` so you have to litter you code with `x === undefined || x === null` this is the one occurrence where `==` is actually helpful, as `(x === undefined || x === null) === (x == null)`

Generally `==` is dangerous and never should be used, however if you are unsure if your getting a null or undefined used `== null`

Reply:
Thanks for the clarification.